### PR TITLE
Add an external cache load balancer and allow https traffic to it.

### PR DESCRIPTION
**Pull request by deanwilson**

### CODE REVIEW

**Comment by surminus** [comment]
We should also add a CNAME for `www-origin` like we have done [here](https://github.com/alphagov/govuk-aws/blob/master/terraform/projects/app-backend-lb/main.tf#L152-L159)

File: terraform/data/app-cache/integration/common.tfvars [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"app_service_records" = [
+  "assets-origin",
+  "draft-origin"
```
Comment (surminus) [inline terraform/data/app-cache/integration/common.tfvars]: missing a comma

Comment (deanwilson) [inline terraform/data/app-cache/integration/common.tfvars]: The last entry in the existing data files does not have a comma. So it's either not allowed or I'm being consistent. Pick an excuse ;)

Comment (surminus) [inline terraform/data/app-cache/integration/common.tfvars]: That's not the last entry though

**Pull request merged**